### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by UNOSweeper** [ldab/ESP32-CAM-Picture-Sharing#4]
Hi, idab. Although my post is not about an issue on your "working example with ESP32-CAM", I need a little more information  how to apply this example.
I have read your Github page "working example with ESP32-CAM" thoroughly with great interest, visited the links provided on it and looked at the contents of your files, because I am very interested in the main.cpp (I pasted this in the Arduino IDE). I want to apply this example for making a bird cam with the ESP32-AI-Thinker cam, powered by batteries.
On your README.md I read that "the idea behind this project is to upload the picture via FTP making it available on an URL something like: www.mysite.com/my_picture.jpg. In your example you chose Blynk. 
However I prefer to have the images on my own website for which I have my own paid domain name such as https://www.mijn-site.nl (not my real domain name).
I don't have enough knowledge to simply replace everything Blynk is mentioned in the main.cpp with something else related to my domain so it will upload the pictures to my website. 
Some changes are simple like the WIFI and FTP server credentials, working directoriy and come up with a pic_name, for example "livecam.jpg", but a pic_url becomes more difficult to guess. Is that for example "/www.mijn-site.nl/livecam.jpg"?
In summary, my question is how to modify main.cpp so that it can also work for any site. Maybe as a second example on your page. Or you could also add that information in the concerning places of your existing main.cpp. 
I am very grateful if you can help me with the relevant information.

**Comment by ldab** [comment]
The way you upload pictures to your own website depends on how it is hosted and how your provider expose the URL path, you may not be allowed to update pictures over FTP, or may need an encrypted FTP (which is not covered on this example).

I'm afraid I already covered the topics you asked, for example:

- When the website is hosted on 000webhostapp, the uploaded picture would be available at `https://my_user.000webhostapp.com/gallery_gen/XxXxXxXx.jpg`
- And on the ESP32 you upload to the FTP path `ftp.ChangeWorkDir("/public_html/zyro/gallery_gen/");`
